src/views/Download.vue

```diff
@@ -23,65 +23,65 @@
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/bukkit.png" alt="Bukkit">
               Bukkit
             </span>
-            <small>For CraftBukkit/Spigot/Paper etc, 1.8.8 or newer</small>
+            <small>Spigot, Paper etc. (1.8.8 to 1.16.4)</small>
           </a>
           <a
             :href="downloads.bungee"
             v-on:click="logDownload('bungee')"
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/bungeecord.png" alt="BungeeCord">
               BungeeCord
             </span>
-            <small>For BungeeCord/Waterfall etc, use latest</small>
+            <small>BungeeCord, Waterfall etc. (latest only)</small>
           </a>
           <a
             :href="downloads.sponge"
             v-on:click="logDownload('sponge')"
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/sponge.png" alt="Sponge">
               Sponge
             </span>
-            <small>For SpongeForge/SpongeVanilla API 5-8</small>
+            <small>SpongeForge &amp; SpongeVanilla (API 5 to 7)</small>
           </a>
           <a
             :href="downloads.nukkit"
             v-on:click="logDownload('nukkit')"
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/nukkit.png" alt="Nukkit">
               Nukkit
             </span>
-            <small>For NukkitX, b93 or newer</small>
+            <small>NukkitX (b93 or newer)</small>
           </a>
           <a
             :href="downloads.velocity"
             v-on:click="logDownload('velocity')"
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/velocity.png" alt="Velocity">
               Velocity
             </span>
-            <small>For Velocity, 1.1.0 or newer</small>
+            <small>Velocity (1.1.0 or newer)</small>
           </a>
           <a
             :href="downloads['bukkit-legacy']"
             v-on:click="logDownload('bukkit-legacy')"
             class="resource"
           >
             <span>
-              <font-awesome icon="arrow-alt-circle-down" />
+              <img src="@/assets/logos/bukkit.png" alt="Bukkit">
               Bukkit Legacy
             </span>
-            <small>For CraftBukkit/Spigot/Paper etc, 1.7.10 only</small>
+            <small>For Spigot, Paper etc. (1.7.10 only)</small>
```

Comment:
You removed the "For" from all others, I'd say it should be removed here as well.